### PR TITLE
LPS-123651 use the method with the extension to provide an unique title to googleDrive files

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**
When you are providing a unique title, you should take in consideration that the final filename (the title plus the extension) should be unique too or an error will be thrown, so now the extension is provided to check the title and the title plus the extension.

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by adolfopa** [comment]
@AliciaGarciaGarcia Can you explain in the pull description what you're trying to do here? I don't understand why the unique title provider should handle filenames as well.

File: modules/apps/document-library-opener/document-library-opener-service/src/main/java/com/liferay/document/library/opener/internal/upload/UniqueFileEntryTitleProviderImpl.java [hunk 1]
```diff
@@ -74,12 +85,42 @@ private boolean _exists(long groupId, long folderId, String fileName) {
...
+			throw new SystemException(portalException);
+		}
+
+		return false;
+	}
+
+	private String _provide(long groupId, long folderId, String title)
+		throws PortalException {
+
+		return _uniqueFileNameProvider.provide(
```
Comment (4lejandrito) [inline modules/apps/document-library-opener/document-library-opener-service/src/main/java/com/liferay/document/library/opener/internal/upload/UniqueFileEntryTitleProviderImpl.java:109]: Hi @AliciaGarciaGarcia, should we check here if the fileName exists too?

Comment (4lejandrito) [reply modules/apps/document-library-opener/document-library-opener-service/src/main/java/com/liferay/document/library/opener/internal/upload/UniqueFileEntryTitleProviderImpl.java:109]: I see we don't have the extension, so it is not possible.

Comment (AliciaGarciaGarcia) [reply modules/apps/document-library-opener/document-library-opener-service/src/main/java/com/liferay/document/library/opener/internal/upload/UniqueFileEntryTitleProviderImpl.java:109]: In this method, without the extension, we cannot check it.
There is no restriction that the filename on one file should not be the title of another (extrange, yes, but not illegal)
We can deprecate the method

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by 4lejandrito** [comment]
See https://github.com/liferay-lima/liferay-portal/pull/1125.

**Pull request closed without merging**